<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fractal_Dialog</class>
 <widget class="QDialog" name="Fractal_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Множина Жюлія</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Icons/Icons/2.png</normaloff>:/Icons/Icons/2.png</iconset>
  </property>
  <widget class="QLabel" name="Constant_C">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Опис, що за що відповідає константа C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Constant C:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PlusLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="ImaginaryLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QLabel" name="Color_Scheme">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Опис кольорової схеми&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Color Scheme:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Scale">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Масштаб&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Scale:</string>
   </property>
  </widget>
  <widget class="QSlider" name="ScaleSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="singleStep">
    <number>25</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="IterationLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Масштаб&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Iteration:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IterationCheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>182</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="IterationSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="RealSpinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>-0.745430000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ImaginarySpinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="value">
    <double>0.113010000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="unitsComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mm</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="singleStep">
    <number>25</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
